app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java

```diff
@@ -42,14 +43,19 @@
     private TextView profileReposCounter;
     private String profileHtmlUrl;
     private RepositoryAdapter repositoriesAdapter;
+    private View viewFragment;
 
     private SubscriberDetailPresenter presenter;
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         loadArgs();
-        repositoriesAdapter = new RepositoryAdapter();
+        if (savedInstanceState == null) {
+            setRetainInstance(true);
```

Comment:
The purpose of this is retain each attribute/field of the fragment to reuse it everytime that a new fragment is created. Actually, this is to reuse mainly the MVP pattern of this fragment (View, Presenter and interactor).